### PR TITLE
Upgrade Crystal 0.28.0

### PR DESCRIPTION
**Pull request by bcardiff**
I found the following changes needed to upgrade to the upcoming Crystal 0.28.0.

If some macro `compare_versions` are added it should be possible to support current versions as well.

Feel free to extract these changes and close the PR if you prefer to handle the migration in a specific way.

### CODE REVIEW

**Comment by bcardiff** [comment]
Note that db related shards are also in the process of been updated for Crystal 0.28.0.

File: src/lucky/params.cr [hunk 1]
```diff
@@ -367,8 +367,9 @@ class Lucky::Params
     multipart_params = {} of String => String
     multipart_files = {} of String => Lucky::UploadedFile
     body_io = IO::Memory.new(body)
-    boundary =
-      HTTP::Multipart.parse_boundary(request.headers["Content-Type"]).to_s
+
+    boundary = MIME::Multipart.parse_boundary(request.headers["Content-Type"]).to_s
```
Comment (jwoertink) [inline src/lucky/params.cr]: Does this need to be required? `require "mime"`

Comment (bcardiff) [reply src/lucky/params.cr]: It could. `MIME` ended up been available because it is required in the `StaticFileHandler` of the stdlib.

Comment (jwoertink) [reply src/lucky/params.cr]: hmm interesting. Yeah, the `Lucky::StaticFileHandlers` inherits from the `HTTP::StaticFileHandler` which does require mime already. Not sure why the CI said mime is undefined...

**Comment by bcardiff** [comment]
Oh, that’s because 0.28 hasn’t been released yet :-)

**Comment by jwoertink** [comment]
hahah 😅 my bad.

**Comment by paulcsmith** [comment]
Closing in favor of https://github.com/luckyframework/lucky/pull/766 which updates the Dockerfile so it passes in CI. Thanks so much for this @bcardiff. It's so helpful that you help us get ready for new releases ❤️

**Comment by bcardiff** [comment]
FYI the crystal-pg/mysql/sqlite3 shards are already released with 0.28.0 support

**Comment by paulcsmith** [comment]
Wonderful :D Thanks for the update

**Pull request closed without merging**